modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -110,15 +110,25 @@ def synchronous
             auto_claim.reload
           end
 
-          render json: ClaimsApi::V2::Blueprints::AutoEstablishedClaimBlueprint.render(
-            auto_claim, root: :data, async: false
-          ), status: :accepted, location: url_for(controller: 'claims', action: 'show', id: auto_claim.id)
+          form_blueprint = JSON.parse(ClaimsApi::V2::Blueprints::AutoEstablishedClaimBlueprint.render(
+                                        auto_claim, root: :data, async: false
+                                      ))
+
+          form_meta = { transactionId: auto_claim.transaction_id }
+
+          render_hash = {}
+          render_hash[:meta] = form_meta if auto_claim.transaction_id.present?
+          render_hash[:data] = form_blueprint['data']
```

Comment:
I'd prefer this to live in the Blueprinter.

Reply:
@FonzMP  Ya I did try that.  The issue I ran into was the [BluePrinter](https://github.com/procore-oss/blueprinter) wants a root object sent in, but there is no root for this.  `Meta` and `Data` are both at the same level.  Perhaps we can look at this and see if I am missing something.

I do think the one thing I got to work using the BluePrinter as having `data` as a root, and then in the controller just doing a return and setting something along the lines of `variable[:data]`  on the return to remove the outer data wrapper, but that felt hacky.